airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryDestinationTest.java

```diff
@@ -227,12 +231,53 @@ void testCheckSuccess(final DatasetIdResetter resetDatasetId) {
   void testCheckFailure(final DatasetIdResetter resetDatasetId) {
     ((ObjectNode) config).put(BigQueryConsts.CONFIG_PROJECT_ID, "fake");
     resetDatasetId.accept(config);
-    final AirbyteConnectionStatus actual = new BigQueryDestination().check(config);
-    final String actualMessage = actual.getMessage();
+
+    // Assert that check throws exception. Later it will be handled by IntegrationRunner
+    final ConfigErrorException ex = assertThrows(ConfigErrorException.class, () -> {
+      new BigQueryDestination().check(config);
+    });
+
+    final String actualMessage = ex.getMessage();
+    LOGGER.info("Checking expected failure message:" + actualMessage);
```

Comment:
Not sure if this `LOGGER` is needed. To keep the test simpler, may suggest just having this as 

```java
assertThat(ex.getMessage()).contains("Access Denied");
```

cc: @ChristopheDuong is there any reason we may want to keep this `LOGGER.info`?

Reply:
> Not sure if this LOGGER is needed. 

probably not? just for surfacing information in the logs